dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunction.cs

```diff
@@ -515,6 +516,16 @@ public AIFunction AsAIFunction(Kernel? kernel = null)
         return new KernelAIFunction(this, kernel);
     }
 
+    /// <summary>Creates an <see cref="KernelFunction"/> for this <see cref="AIFunction"/>.</summary>
+    /// <param name="aiFunction">The <see cref="AIFunction"/> instance to wrap in a <see cref="KernelFunction"/>.</param>
+    /// <returns>An instance of <see cref="KernelFunction"/> that, when invoked, will in turn invoke the current <see cref="AIFunction"/>.</returns>
+    [Experimental("SKEXP0001")]
+    public static KernelFunction FromAIFunction(AIFunction aiFunction)
```

Comment:
I would follow the same pattern here:

https://github.com/microsoft/semantic-kernel/blob/e96c693851a2e4be14a9e7fd835c4087b063dc58/dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunction.cs#L513

Basically suggest adding a `AIFunctionExtensions.cs` where

```csharp
namespace Microsoft.SemanticKernel;

public static class AIFunctionExtensions 
{
    public static KernelFunction AsKernelFunction(this AIFunction aiFunctin)
    {
        Verify.NotNull(aiFunction);
        return new AIFunctionKernelFunction(aiFunction);
    }
}

```